src/mscorlib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -1060,7 +1093,7 @@ public static IntPtr GetHINSTANCE(Module m)
             }
 
             if (rtModule == null)
-                throw new ArgumentNullException("m", Environment.GetResourceString("Argument_MustBeRuntimeModule"));
+                throw new ArgumentNullException(Environment.GetResourceString("Argument_MustBeRuntimeModule"));
```

Comment:
@yizhang82 that's weird , it's merge conflict and git just over-wrote it and did not see it as a merge conflict. This change to use "m" was from the previous PR ( Hugh Bellamy)